utilities/unittests/test_storage.py

```
@@ -169,3 +170,213 @@ class TestConstructDatavolumeSourceDictUnsupported:
+        mock_data_volume_class.return_value.__exit__ = MagicMock(return_value=None)
+        return mock_dv
+
+    @patch("utilities.storage.sc_volume_binding_mode_is_wffc", return_value=False)
+    @patch("utilities.storage.DataVolume")
+    @patch("utilities.storage.validate_file_exists_in_url")
+    @patch("utilities.infra.url_excluded_from_validation", return_value=True)
+    @patch("utilities.storage.artifactory_credentials")
+    def test_create_dv_creates_both_artifactory_resources(
+        self,
+        mock_artifactory_credentials,
+        mock_excluded,
```

Ruff (0.16.0)

[warning] Unused method argument: `mock_excluded`

(ARG002)